app/src/routes/__layout.svelte

```diff
@@ -1,9 +1,10 @@
 <script lang="ts">
-	import { onMount } from 'svelte';
-	import { clusterApiUrl } from '@solana/web3.js';
-	import WalletProvider from '$lib/WalletProvider.svelte';
 	import ConnectionProvider from '$lib/ConnectionProvider.svelte';
+	import WalletProvider from '$lib/WalletProvider.svelte';
+	import { clusterApiUrl } from '@solana/web3.js';
+	import { onMount } from 'svelte';
 	import idl from '../../../target/idl/solana_svelte_counter.json';
+	import '../styles.css';
```

Comment:
For testing purpose this is ok, but we need to thing in how to use these styles without importing in layout. Probably we need to use `:global()`